### PR TITLE
Create tabs for user profile, messages, and calendar

### PR DESCRIPTION
**Pull request by glkidd**
I worked on creating the tabs for the home page. These included the user profile, messages, and calendar. I used the react bootstrap Tabs so it should be responsive. On the prototype we didn't have an icon for the user profile tab so I added one because I felt it looked more consistent with the other tabs. 

Screen Shot 2021-01-20 at 11 10 59 PM

### CODE REVIEW

File: cloud-haven-react-app/src/components/TabSystem/TabSystem.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.Tabs > .nav-link:hover {
+    color: black;
+    border-radius: 20px;
+    border: none;
+}
+
+.TabIcon {
+    position: relative;
+    bottom: .1em;
+}
```
Comment (Michelle5102) [inline cloud-haven-react-app/src/components/TabSystem/TabSystem.css:32]:
```suggestion
}

```
Newline character :)

File: cloud-haven-react-app/src/components/TabSystem/TabSystem.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                Messages goes here. 
+            </Tab>
+            <Tab eventKey="Calendar" title={<span> <img className="TabIcon" src={calendar} height="30"/> Calendar </span>}>
+                Calendar goes here.
+            </Tab>
+        </Tabs>
+    )
+}
+
+export default TabSystem;
```
Comment (Michelle5102) [inline cloud-haven-react-app/src/components/TabSystem/TabSystem.js:24]:
```suggestion
export default TabSystem;

```
Newline character

**Review by Michelle5102 — CHANGES_REQUESTED** [review]
Just some missing newlines at the end of files!

Also, we'll need the tabs system to be dynamic (we're going to need different tabs depending on application we're in and which features users click on). I'm not opposed to merging this pretty much as is, but maybe we should have a group discussion about how we want to structure the website and how everything interacts.

**Comment by Michelle5102** [comment]
Also, just ran it, and it looks good! :)

**Review by patrickbartlett — APPROVED** [review]
It all looks good, Michelle raised a good point on whether or not the tabs will need to be dynamic, we should that with the group.

**Review by Michelle5102 — APPROVED** [review]

**Pull request merged**